src/tools/kata-ctl/src/check.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+
+fn check_cpu() -> Result<()> {
+    println!("Check CPU not implemented");
```

Comment:
You could use the built-in unimplemented macro for these functions:

```suggestion
    unimplemented!("Check CPU not implemented");
```